app/models/manageiq/providers/openstack/network_manager/cloud_network.rb

```diff
@@ -1,15 +1,23 @@
 class ManageIQ::Providers::Openstack::NetworkManager::CloudNetwork < ::CloudNetwork
+  include ProviderObjectMixin
+  include AsyncDeleteMixin
+  include SupportsFeatureMixin
+
   require_nested :Private
   require_nested :Public
 
+  supports :update_cloud_network
+  supports :create_cloud_network
+  supports :delete_cloud_network
+
   def self.remapping(options)
     new_options = options.dup
     new_options[:router_external] = options[:external_facing] if options[:external_facing]
     new_options.delete(:external_facing)
     new_options
   end
 
-  def self.raw_create_network(ext_management_system, options)
+  def self.create_cloud_network(ext_management_system, options)
```

Comment:
yeah, I was just thinking about this, this PR does more. I moves also the factory from the base class to the Manager class. So we should probably keep all of that, since that is the pattern elsewhere. But we can start a thread to chat about that